docs/tutorials/simple_prf_simulated.ipynb

```diff
@@ -0,0 +1,2328 @@
+    "evaluate 20 parameter combinations at the same time (which saves us some memory)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "f3e59349",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Processing parameter grid chunks: 100%|██████████| 50/50 [00:02<00:00, 24.09it/s, loss=116]  \n"
+     ]
+    }
+   ],
+   "source": [
+    "from prfmodel.fitters.grid import GridFitter\n",
+    "\n",
+    "grid_fitter = GridFitter(\n",
+    "    model=prf_model,\n",
+    "    stimulus=stimulus,\n",
+    ")\n",
+    "\n",
+    "grid_history, grid_params = grid_fitter.fit(\n",
+    "    data=simulated_response,\n",
+    "    parameter_values=param_ranges,\n",
+    "    chunk_size=20,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "d9e83b64",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>mu_x</th>\n",
+       "      <th>mu_y</th>\n",
+       "      <th>sigma</th>\n",
+       "      <th>shape_1</th>\n",
+       "      <th>rate_1</th>\n",
+       "      <th>shape_2</th>\n",
+       "      <th>rate_2</th>\n",
+       "      <th>weight</th>\n",
+       "      <th>baseline</th>\n",
+       "      <th>amplitude</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>-2.333333</td>\n",
+       "      <td>1.666667</td>\n",
+       "      <td>1.888889</td>\n",
+       "      <td>6.0</td>\n",
+       "      <td>0.9</td>\n",
+       "      <td>12.0</td>\n",
+       "      <td>0.9</td>\n",
+       "      <td>0.35</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>1.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       mu_x      mu_y     sigma  shape_1  rate_1  shape_2  rate_2  weight  \\\n",
+       "0 -2.333333  1.666667  1.888889      6.0     0.9     12.0     0.9    0.35   \n",
+       "\n",
+       "   baseline  amplitude  \n",
+       "0       0.0        1.0  "
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "grid_params"
```

Comment:
when viewing the tutorial in dark mode, the dataframe headers are not visible. might be an issue in the theme we're using